openupgrade_scripts/scripts/mail/15.0.1.5/post-migration.py

```diff
@@ -103,8 +103,9 @@ def repl_placeholder(match):
 def _migrate_placeholder_html(string):
     """
     Replace dynamic placeholders in HTML fields:
-    Example: 'Your name is ${object.name}' -> 'Your name is <t t-out="object.name"></t>'
+    Example: 'Your name is ${object.name | safe}' -> 'Your name is <t t-out="object.name"></t>'
     """
+    string = re.sub(r"\s?\|\s?safe\s?", "", string)
```

Comment:
there can be much more [filters](https://jinja.palletsprojects.com/en/stable/templates/#filters) - could you extend the regex to just replace anything after the first pipe?